Library/Formula/gcc.rb

```diff
@@ -32,6 +32,15 @@ def osmajor
     sha256 "77bce635f78bc26bd01010b5ece480251af223bf2dba6d48c29af6b29b441296" => :mountain_lion
   end
 
+  if MacOS::Xcode.version > "7.0" || MacOS::CLT.version > "7.0"
```

Comment:
This line will fail on systems where one of the versions is nil (i.e., one or the other isn't installed).

I think we can just apply the patch unconditionally, though.